sys-config/ltoize/files/package.cflags/ltoworkarounds.conf

```diff
@@ -266,8 +267,7 @@ net-libs/libtorrent-rasterbar *FLAGS-="-mtls-dialect=gnu2" # causes memory corru
 # END: TLS dialect workarounds
 
 # BEGIN: Semantic Interposition workarounds
-app-crypt/tpm2-tss *FLAGS-="${SEMINTERPOS}" # Problem with inlining, see issue #592
-app-misc/tracker *FLAGS-="${SEMINTERPOS}" # builds but makes gnome-base/nautilus deadlock
```

Comment:
I think you missed removing the above comment

Reply:
oops, idk why those 2 things got deleted, let me fix that rq.

Reply:
Fixed!